app/repositories/submission_repository.rb

```diff
@@ -1,18 +1,26 @@
 class SubmissionRepository
   def rejected
-    Submission.where(rejected: true)
+    Submission.where(rejected: true).order('created_at DESC')
   end
 
   def valid
     Submission.where(rejected: false)
```

Comment:
I talked with @szynszyliszys, it should be `created_at ASC`everywhere except the results.